### ISSUE TITLE
My store continues to reject orders and show errors on the page.

### ISSUE THREAD

**Issue by virgocar242** [woothemes/woocommerce#32008]
### Prerequisites

- [ ] I have carried out troubleshooting steps and I believe I have found a bug.
- [X] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

When I'm filling out my vehicle registration info for my website, there is a continuous error that shows up on the page. Which this is happening orders are coming directly through but are being cancelled.

### Expected behavior

When I went to fill out the information, I expected the order to process as normal.

### Actual behavior

A green square popped up around a certain field and did not allow the order to process on the customers end.

### Steps to reproduce

![IMG_8114](https://user-images.githubusercontent.com/100740850/156280616-61d259f1-0fd1-4bd8-9467-227ef8e6eca1.jpg)
![IMG_8115](https://user-images.githubusercontent.com/100740850/156280744-f86dd6b1-cb25-4d56-970c-b53dbc267903.jpg)

### WordPress Environment

`
### WordPress Environment ###

WordPress address (URL): https://www.virgocarrental.com
Site address (URL): https://www.virgocarrental.com
WC Version: 5.9.0
REST API Version: ✔ 5.9.0
WC Blocks Version: ✔ 6.1.0
Action Scheduler Version: ✔ 3.3.0
WC Admin Version: ✔ 2.8.0
Log Directory Writable: ✔
WP Version: 5.9.1
WP Multisite: –
WP Memory Limit: 3 GB
WP Debug Mode: –
WP Cron: ✔
Language: en_US
External object cache: –

### Server Environment ###

Server Info: Apache
PHP Version: 8.0.16
PHP Post Max Size: 516 MB
PHP Time Limit: 30
PHP Max Input Vars: 1000
cURL Version: 7.81.0
OpenSSL/1.1.1m

SUHOSIN Installed: –
MySQL Version: 5.7.23-23
Max Upload Size: 512 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 5.9.0
WC Database Prefix: wp_lcgf_
Total Database Size: 81.03MB
Database Data Size: 60.01MB
Database Index Size: 21.02MB
wp_lcgf_woocommerce_sessions: Data: 2.02MB + Index: 0.02MB + Engine InnoDB
wp_lcgf_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_lcgf_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_lcgf_woocommerce_downloadable_product_permissions: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_lcgf_woocommerce_order_items: Data: 0.06MB + Index: 0.02MB + Engine InnoDB
wp_lcgf_woocommerce_order_itemmeta: Data: 0.25MB + Index: 0.27MB + Engine InnoDB
wp_lcgf_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_lcgf_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_lcgf_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_lcgf_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_lcgf_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_lcgf_woocommerce_payment_tokens: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_lcgf_woocommerce_payment_tokenmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_lcgf_woocommerce_log: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
dcb8fb_wp_lcgf_alc_chat: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
dcb8fb_wp_lcgf_alc_departments: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
dcb8fb_wp_lcgf_alc_users: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
dcb8fb_wp_lcgf_ariadminer_connections: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
dcb8fb_wp_lcgf_car_rental_benefits: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
dcb8fb_wp_lcgf_car_rental_body_types: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
dcb8fb_wp_lcgf_car_rental_bookings: Data: 0.16MB + Index: 0.22MB + Engine InnoDB
dcb8fb_wp_lcgf_car_rental_booking_options: Data: 0.09MB + Index: 0.20MB + Engine InnoDB
dcb8fb_wp_lcgf_car_rental_closed_dates: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
dcb8fb_wp_lcgf_car_rental_customers: Data: 0.17MB + Index: 0.09MB + Engine InnoDB
dcb8fb_wp_lcgf_car_rental_discounts: Data: 0.02MB + Index: 0.11MB + Engine InnoDB
dcb8fb_wp_lcgf_car_rental_distances: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
dcb8fb_wp_lcgf_car_rental_emails: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
dcb8fb_wp_lcgf_car_rental_extras: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
dcb8fb_wp_lcgf_car_rental_features: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
dcb8fb_wp_lcgf_car_rental_fuel_types: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
dcb8fb_wp_lcgf_car_rental_invoices: Data: 16.52MB + Index: 0.02MB + Engine InnoDB
dcb8fb_wp_lcgf_car_rental_items: Data: 0.02MB + Index: 0.25MB + Engine InnoDB
dcb8fb_wp_lcgf_car_rental_item_features: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
dcb8fb_wp_lcgf_car_rental_item_locations: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
dcb8fb_wp_lcgf_car_rental_locations: Data: 0.02MB + Index: 0.13MB + Engine InnoDB
dcb8fb_wp_lcgf_car_rental_logs: Data: 0.02MB + Index: 0.16MB + Engine InnoDB
dcb8fb_wp_lcgf_car_rental_manufacturers: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
dcb8fb_wp_lcgf_car_rental_options: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
dcb8fb_wp_lcgf_car_rental_payment_methods: Data: 0.02MB + Index: 0.11MB + Engine InnoDB
dcb8fb_wp_lcgf_car_rental_prepayments: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
dcb8fb_wp_lcgf_car_rental_price_groups: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
dcb8fb_wp_lcgf_car_rental_price_plans: Data: 0.02MB + Index: 0.08MB + Engine InnoDB
dcb8fb_wp_lcgf_car_rental_settings: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
dcb8fb_wp_lcgf_car_rental_taxes: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
dcb8fb_wp_lcgf_car_rental_transmission_types: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
dcb8fb_wp_lcgf_commentmeta: Data: 0.05MB + Index: 0.03MB + Engine MyISAM
dcb8fb_wp_lcgf_comments: Data: 0.10MB + Index: 0.02MB + Engine MyISAM
dcb8fb_wp_lcgf_ewwwio_images: Data: 0.07MB + Index: 0.05MB + Engine MyISAM
dcb8fb_wp_lcgf_layerslider: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
dcb8fb_wp_lcgf_links: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
dcb8fb_wp_lcgf_options: Data: 0.63MB + Index: 0.05MB + Engine MyISAM
dcb8fb_wp_lcgf_popularpostsdata: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
dcb8fb_wp_lcgf_popularpostssummary: Data: 1.52MB + Index: 4.95MB + Engine InnoDB
dcb8fb_wp_lcgf_postmeta: Data: 0.27MB + Index: 0.15MB + Engine MyISAM
dcb8fb_wp_lcgf_posts: Data: 0.56MB + Index: 0.04MB + Engine MyISAM
dcb8fb_wp_lcgf_revslider_css: Data: 0.09MB + Index: 0.00MB + Engine MyISAM
dcb8fb_wp_lcgf_revslider_layer_animations: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
dcb8fb_wp_lcgf_revslider_navigations: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
dcb8fb_wp_lcgf_revslider_sliders: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
dcb8fb_wp_lcgf_revslider_slides: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
dcb8fb_wp_lcgf_revslider_static_slides: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
dcb8fb_wp_lcgf_termmeta: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
dcb8fb_wp_lcgf_terms: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
dcb8fb_wp_lcgf_term_relationships: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
dcb8fb_wp_lcgf_term_taxonomy: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
dcb8fb_wp_lcgf_usermeta: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
dcb8fb_wp_lcgf_users: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
dcb8fb_wp_lcgf_wpfm_backup: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
dcb8fb_wp_lcgf_wplc_chat_msgs: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
dcb8fb_wp_lcgf_wplc_chat_sessions: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
dcb8fb_wp_lcgf_wplc_offline_messages: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_lcgf_actionscheduler_actions: Data: 0.44MB + Index: 0.52MB + Engine InnoDB
wp_lcgf_actionscheduler_claims: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_lcgf_actionscheduler_groups: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_lcgf_actionscheduler_logs: Data: 0.30MB + Index: 0.23MB + Engine InnoDB
wp_lcgf_aioseo_cache: Data: 0.17MB + Index: 0.03MB + Engine InnoDB
wp_lcgf_aioseo_notifications: Data: 0.03MB + Index: 0.06MB + Engine InnoDB
wp_lcgf_aioseo_posts: Data: 0.19MB + Index: 0.02MB + Engine InnoDB
wp_lcgf_alc_chat: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_lcgf_alc_departments: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_lcgf_alc_users: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_lcgf_ariadminer_connections: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_lcgf_car_rental_benefits: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_lcgf_car_rental_body_types: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_lcgf_car_rental_bookings: Data: 0.14MB + Index: 0.22MB + Engine InnoDB
wp_lcgf_car_rental_booking_options: Data: 0.08MB + Index: 0.20MB + Engine InnoDB
wp_lcgf_car_rental_closed_dates: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_lcgf_car_rental_customers: Data: 0.17MB + Index: 0.09MB + Engine InnoDB
wp_lcgf_car_rental_discounts: Data: 0.02MB + Index: 0.11MB + Engine InnoDB
wp_lcgf_car_rental_distances: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_lcgf_car_rental_emails: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_lcgf_car_rental_extras: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_lcgf_car_rental_features: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_lcgf_car_rental_fuel_types: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_lcgf_car_rental_invoices: Data: 16.52MB + Index: 0.02MB + Engine InnoDB
wp_lcgf_car_rental_items: Data: 0.02MB + Index: 0.25MB + Engine InnoDB
wp_lcgf_car_rental_item_features: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_lcgf_car_rental_item_locations: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_lcgf_car_rental_locations: Data: 0.02MB + Index: 0.13MB + Engine InnoDB
wp_lcgf_car_rental_logs: Data: 0.02MB + Index: 0.16MB + Engine InnoDB
wp_lcgf_car_rental_manufacturers: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_lcgf_car_rental_options: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_lcgf_car_rental_payment_methods: Data: 0.02MB + Index: 0.11MB + Engine InnoDB
wp_lcgf_car_rental_prepayments: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_lcgf_car_rental_price_groups: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_lcgf_car_rental_price_plans: Data: 0.02MB + Index: 0.08MB + Engine InnoDB
wp_lcgf_car_rental_settings: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_lcgf_car_rental_taxes: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_lcgf_car_rental_transmission_types: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_lcgf_ce4wp_abandoned_checkout: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_lcgf_ce4wp_contacts: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_lcgf_commentmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_lcgf_comments: Data: 0.13MB + Index: 0.09MB + Engine InnoDB
wp_lcgf_cscs_db_subscriptions: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_lcgf_defender_audit_log: Data: 0.02MB + Index: 0.08MB + Engine InnoDB
wp_lcgf_defender_email_log: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_lcgf_defender_lockout: Data: 0.31MB + Index: 0.39MB + Engine InnoDB
wp_lcgf_defender_lockout_log: Data: 0.36MB + Index: 0.22MB + Engine InnoDB
wp_lcgf_defender_scan: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_lcgf_defender_scan_item: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_lcgf_ewwwio_images: Data: 0.07MB + Index: 0.05MB + Engine MyISAM
wp_lcgf_layerslider: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_lcgf_links: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_lcgf_mailchimp_carts: Data: 0.05MB + Index: 0.00MB + Engine InnoDB
wp_lcgf_mailchimp_jobs: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_lcgf_options: Data: 7.25MB + Index: 0.20MB + Engine InnoDB
wp_lcgf_popularpostsdata: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_lcgf_popularpostssummary: Data: 1.52MB + Index: 2.88MB + Engine InnoDB
wp_lcgf_postmeta: Data: 2.50MB + Index: 1.91MB + Engine InnoDB
wp_lcgf_posts: Data: 2.48MB + Index: 0.13MB + Engine InnoDB
wp_lcgf_redirection_404: Data: 0.17MB + Index: 0.11MB + Engine InnoDB
wp_lcgf_redirection_groups: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_lcgf_redirection_items: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_lcgf_redirection_logs: Data: 0.16MB + Index: 0.05MB + Engine InnoDB
wp_lcgf_rental_price_date: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_lcgf_revslider_css: Data: 0.14MB + Index: 0.02MB + Engine InnoDB
wp_lcgf_revslider_css_bkp: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_lcgf_revslider_layer_animations: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_lcgf_revslider_layer_animations_bkp: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_lcgf_revslider_navigations: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_lcgf_revslider_navigations_bkp: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_lcgf_revslider_sliders: Data: 0.05MB + Index: 0.02MB + Engine InnoDB
wp_lcgf_revslider_sliders_bkp: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_lcgf_revslider_slides: Data: 0.06MB + Index: 0.02MB + Engine InnoDB
wp_lcgf_revslider_slides_bkp: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_lcgf_revslider_static_slides: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_lcgf_revslider_static_slides_bkp: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_lcgf_termmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_lcgf_terms: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_lcgf_term_relationships: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_lcgf_term_taxonomy: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_lcgf_usermeta: Data: 0.17MB + Index: 0.14MB + Engine InnoDB
wp_lcgf_users: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_lcgf_wc_admin_notes: Data: 0.06MB + Index: 0.00MB + Engine InnoDB
wp_lcgf_wc_admin_note_actions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_lcgf_wc_cancel_orders: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_lcgf_wc_category_lookup: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_lcgf_wc_customer_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_lcgf_wc_download_log: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_lcgf_wc_order_coupon_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_lcgf_wc_order_product_lookup: Data: 0.08MB + Index: 0.06MB + Engine InnoDB
wp_lcgf_wc_order_stats: Data: 0.05MB + Index: 0.05MB + Engine InnoDB
wp_lcgf_wc_order_tax_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_lcgf_wc_product_meta_lookup: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_lcgf_wc_reserved_stock: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_lcgf_wc_tax_rate_classes: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_lcgf_wc_webhooks: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_lcgf_wpfm_backup: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_lcgf_wpforms_tasks_meta: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_lcgf_wplc_chat_msgs: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_lcgf_wplc_chat_sessions: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_lcgf_wplc_offline_messages: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_lcgf_wpmailsmtp_debug_events: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_lcgf_wpmailsmtp_tasks_meta: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_lcgf_wsal_metadata: Data: 1.52MB + Index: 3.03MB + Engine InnoDB
wp_lcgf_wsal_occurrences: Data: 0.08MB + Index: 0.13MB + Engine InnoDB

### Post Type Counts ###

attachment: 132
br_force_sell: 4
customize_changeset: 2
mc4wp-form: 1
nav_menu_item: 7
page: 17
post: 174
product: 23
product_variation: 16
revision: 65
shop_coupon: 1
shop_order: 210
sidebar: 1
spt_shortcodes: 1
spt_testimonial: 4
stm_office: 3
wpcf7_contact_form: 3
wpforms: 1
xlwcty_thankyou: 1

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (40) ###

AddToAny Share Buttons: by AddToAny – 1.8.2
All in One SEO: by All in One SEO Team – 4.1.5.3
All-in-One WP Migration: by ServMask – 7.50
Bluehost: by Bluehost – 2.8.5
Breadcrumb NavXT: by John Havlik – 6.6.0
Checkout Files Upload for WooCommerce: by WP Wham – 2.1.1
Classic Widgets: by WordPress Contributors – 0.2
CMP - Coming Soon & Maintenance Plugin: by NiteoThemes – 4.0.18
Contact Form 7: by Takayuki Miyoshi – 5.5.3
Controlled Admin Access: by WPRuby – 1.5.12
Crisp: by Crisp – 0.30
Custom Elementor Icons: by StylemixThemes – 1.0.2
Custom Icons by Stylemixthemes: by Stylemix Themes – 1.7
Defender: by WPMU DEV – 2.6.5
Duplicate Page: by mndpsingh287 – 4.4.6
Force Sell for WooCommerce: by BeRocket – 3.5.7.2
Jetpack: by Automattic – 10.4
WPBakery Page Builder: by Michael M - WPBakery.com – 6.6.0
Kadence WooCommerce Email Designer: by Kadence WP – 1.4.9
Mailchimp for WooCommerce: by Mailchimp – 2.5.4
MC4WP: Mailchimp for WordPress: by ibericode – 4.8.6
Really Simple SSL: by Really Simple Plugins – 5.2.0
Redirection: by John Godley – 5.1.3
Slider Revolution: by ThemePunch – 6.5.9
SmartCrawl: by WPMU DEV – 2.16.0
Sold Out Badge for WooCommerce: by Charlie Etienne – 2.0.10
STM Motors Extends: by StylemixThemes – 1.6
STM Post Type: by Stylemix Themes – 4.8
STM Importer: by Stylemix Themes – 4.8.8
Motors – Car Dealer, Classifieds & Listing: by StylemixThemes – 6.8.7
Real Testimonials: by ShapedPlugin – 2.4.2
WC Cancel Order: by Vikram S – 3.1.2
Checkout Fields Manager for WooCommerce: by QuadLayers – 5.5.2
WooCommerce: by Automattic – 5.9.0
WP Activity Log for WooCommerce: by WP White Security – 1.3.1
WP Mail SMTP: by WPForms – 3.2.1
WP Rocket: by WP Rocket – 2.8.11
WP Activity Log for WPForms: by WP White Security – 1.1.2
WP Activity Log: by WP White Security – 4.3.3.1
WPForms Lite: by WPForms – 1.7.1.2

### Inactive Plugins (19) ###

Add Logo to Admin: by c.bavota – 1.6.2
Akismet Anti-Spam: by Automattic – 4.2.1
BackWPup: by Inpsyde GmbH – 3.3.4
Better Font Awesome: by MIGHTYminnow & Mickey Kay – 1.6.0
Car Rental System: by Kęstutis Matuliauskas – 5.0
Easy Coming Soon: by Ankit Agarwal
Priyanshu Mittal – 1.9.2

Easy FancyBox: by RavanH – 1.5.7
EWWW Image Optimizer: by Shane Bishop – 3.1.1
Fusion Core: by ThemeFusion – 2.0.3
Increase Maximum Upload File Size: by Imagify – 2.0.4
Kanoo Payment Gateway: by CaribPay (Bahamas) Ltd. – 1.0.0
MOJO Marketplace: by Mike Hansen – 1.0.0
WaspThemes - Yellow Pencil Pro: by WaspThemes – 5.4.7
Widget Importer & Exporter: by Steven Gliebe – 1.3.1
WooCommerce PayPal Checkout Gateway: by WooCommerce – 2.1.3
WordPress Popular Posts: by Hector Cabrera – 3.3.4
WP Fastest Cache: by Emre Vona – 0.9.5
WP File Manager: by mndpsingh287 – 7.1.2
WP Reset: by WebFactory Ltd – 1.93

### Dropin Plugins (2) ###

advanced-cache.php: advanced-cache.php
install.php: install.php

### Must Use Plugins (1) ###

SSO: by Garth Mortensen
Mike Hansen – 0.3

### Settings ###

API Enabled: –
Force SSL: –
Currency: BSD ($)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: car_option (car_option)
composite (composite)
external (external)
grouped (grouped)
simple (simple)
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: –

### WC Pages ###

Shop base: #172 - /reservation/
Cart: #169 - /cart-2/
Checkout: #170 - /checkout-2/
My account: #171 - /my-account-2/
Terms and conditions: #359 - /terms/

### Theme ###

Name: Motors - Child Theme
Version: 4.0
Author URL: http://stylemixthemes.com/
Child Theme: ✔
Parent Theme Name: Motors
Parent Theme Version: 5.0.9
Parent Theme Author URL: https://stylemixthemes.com/
WooCommerce Support: ✔

### Templates ###

Archive Template: Your theme has a woocommerce.php file
you will not be able to override the woocommerce/archive-product.php custom template since woocommerce.php has priority over archive-product.php. This is intended to prevent display issues.

Overrides: motors/woocommerce/cart/cart-totals.php
motors/woocommerce/cart/cart.php
motors/woocommerce/checkout/form-billing.php
motors/woocommerce/checkout/form-checkout.php
motors/woocommerce/checkout/form-shipping.php
motors/woocommerce/checkout/payment-method.php
motors/woocommerce/checkout/payment.php
motors/woocommerce/checkout/review-order.php
motors/woocommerce/checkout/thankyou.php
motors/woocommerce/content-product.php
motors/woocommerce/content-single-product.php
motors/woocommerce/content-widget-product.php
motors/woocommerce/content-widget-reviews.php
motors/woocommerce/emails/email-order-details.php
motors/woocommerce/global/breadcrumb.php
motors/woocommerce/global/quantity-input.php
motors/woocommerce/loop/pagination.php
motors/woocommerce/myaccount/dashboard.php
motors/woocommerce/myaccount/form-edit-account.php
motors/woocommerce/myaccount/form-login.php
motors/woocommerce/myaccount/form-lost-password.php
motors/woocommerce/myaccount/form-reset-password.php
motors/woocommerce/myaccount/lost-password-confirmation.php
motors/woocommerce/myaccount/my-account.php
motors/woocommerce/myaccount/my-address.php
motors/woocommerce/myaccount/my-downloads.php
motors/woocommerce/myaccount/my-orders.php
motors/woocommerce/myaccount/navigation.php
motors/woocommerce/myaccount/orders.php
motors/woocommerce/myaccount/view-order.php
motors/woocommerce/notices/error.php
motors/woocommerce/notices/notice.php
motors/woocommerce/notices/success.php
motors/woocommerce/order/form-tracking.php
motors/woocommerce/order/order-again.php
motors/woocommerce/order/order-details-customer.php
motors/woocommerce/order/order-details-item.php
motors/woocommerce/order/order-details.php
motors/woocommerce/order/tracking.php
motors/woocommerce/single-product/related.php
motors/woocommerce/single-product/review.php
motors/woocommerce/single-product/tabs/additional-information.php
motors/woocommerce/single-product/tabs/description.php
motors/woocommerce/single-product/tabs/tabs.php
motors/woocommerce/single-product-reviews.php

### Checkout Files Upload Settings ###

WooCommerce Checkout Files Upload: no
Total file uploaders: 1
Form extras: no
Enable alert on successful file upload: no
Enable alert on successful file remove: no
Max file size: 0
alg_checkout_files_upload_max_file_size_exceeded_message: Allowed file size exceeded (maximum %max_file_size% MB).
Attach files to admin's new order emails: yes
Attach files to customer's processing order emails: no
Send additional email to admin on user actions: Array
(
)
Attach file on upload action: yes
Recipient(s): (set)
Subject: [{site_title}] Checkout files upload action in customer order ({order_number}) - {order_date}
Email heading: {action}: File #{file_num}
Email content: File name: {file_name}
Action: File removed: File removed
Action: File uploaded: File uploaded
Before: Label:
					 
					%field_label%%required_html%
				
												
					Field Container:
					 
					%button_html% %field_html%
				
												
					Uploaded File:
					 
					%image% %file_name% %remove_button%
				
												
					After:
Label: 
%field_label%%required_html%: 
Field Container: 
%button_html% %field_html%: 
Uploaded File: %image% %file_name% %remove_button%
After: 
"Delete" button style: color:red;
Image style: width:64px;
Enable/disable: yes
Required: no
Accepted file types: 
Validate image dimensions: 
Width: 1
Height: 1
Checkout page: woocommerce_before_checkout_form
Position order (i.e. priority): 1
"Thank You" page: yes
"My Account" page: yes
Label: UPLOAD DRIVER'S LICENSE HERE*
Upload button (single): Choose File
Wrong file type: Wrong file type: "%s"!
Wrong image dimensions: Wrong image dimensions: "%s"! Current: %current_width% x %current_height%. Required: %required_width% x %required_height%.
Couldn't get image dimensions: Couldn't get image dimensions: "%s"!
File is required: Drivers license must be uploaded. (button below Place order)
File was successfully uploaded: File "%s" was successfully uploaded.
File was successfully removed: File "%s" was successfully removed.
Require products: Array
(
)
Require product categories: Array
(
)
Require product tags: Array
(
)
Label: 
%field_label%%required_html%: 
Field Container: 
%button_html% %field_html%: 
Uploaded File: %image% %file_name% %remove_button%
After: 

### Action Scheduler ###

Canceled: 2
Oldest: 2022-02-25 16:08:09 +0000
Newest: 2022-02-27 23:22:12 +0000

Complete: 1,235
Oldest: 2022-01-30 02:34:45 +0000
Newest: 2022-03-02 02:04:04 +0000

Pending: 10
Oldest: 2022-03-02 02:19:04 +0000
Newest: 2022-03-08 15:16:04 +0000

### Status report information ###

Generated at: 2022-03-01 22:08:03 -04:00
`

### Isolating the problem

- [ ] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [ ] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [ ] I can reproduce this bug consistently using the steps above.

**Comment by claudiosanches** [comment]
Hi @virgocar242,

While our goal is to address all the issues reported in this repository, GitHub should be treated as a place to report confirmed bugs only.

The type of issue you submitted looks like a custom code related question. Due to the number of bug reports being submitted, we are prioritizing work on them and are not able to attend to custom code questions. 

The best place to get help further is in the [WooCommerce Community Slack](https://woocommerce.com/community-slack/) in the `#developers` channel. 

If you confirm the bug, please open a new issue and provide us with clear steps to reproduce it, but by accessing your website I'm seeing an error in the browser console on almost all pages of your store that I think may be the cause of this issue for you.

![Screenshot from 2022-03-09 17-15-30](https://user-images.githubusercontent.com/1264099/157527887-0caabcc4-2be2-446e-a821-e4186a968fb9.jpg)
![Screenshot from 2022-03-09 17-15-53](https://user-images.githubusercontent.com/1264099/157527902-a2ef75ac-b863-4220-9f6c-fdd6e80e91c7.jpg)

When there's issues with JavaScript like that it will stop all other JavaScript code that suppose to be loaded after to not load, that's why you are experiencing this issue, there's some 3rd party code stopping WooCommerce to run properly. Note that this is a custom code not included by WooCommerce, I advice you to try find what plugin/theme/snippet introduced it in order to remove this code or fix.

I am closing this issue for now. If I missed something, please leave a comment and we will take a second look.

**Comment by woocommercebot** [comment]
Hi @virgocar242,

  While our goal is to address all the issues reported in this repository,   GitHub should be treated as a place to report confirmed bugs only.

  The type of issue you submitted looks like a support request which may or may not reveal a bug once proper   troubleshooting is done.  In order to confirm the bug, please follow one of the steps below:

  - Review [WooCommerce Self-Service Guide](https://docs.woocommerce.com/document/woocommerce-self-service-guide/)   to see if the solutions listed there apply to your case;
  - If you are a paying customer of WooCommerce, contact WooCommerce support by   [opening a ticket or starting a live chat](https://woocommerce.com/contact-us/);
  - Make a post on [WooCommerce community forum](https://wordpress.org/support/plugin/woocommerce/)

  If you confirm the bug, please provide us with clear steps to reproduce it.

  We are closing this issue for now as it seems to be a support request and not a bug.   If we missed something, please leave a comment and we will take a second look.
